sopel_modules/github/webhook.py

```diff
@@ -75,15 +83,55 @@ def create_table(bot, c):
 
 
 def shutdown_webhook(sopel):
-    global sopel_instance
-    sopel_instance = None
     if 'gh_webhook_server' in sopel.memory:
         print('Stopping webhook server')
         sopel.memory['gh_webhook_server'].stop()
         sopel.memory['gh_webhook_thread'].join()
         print('GitHub webhook shutdown complete')
 
 
+class SopelMiddleware(object):
+    """WSGI Middlware that injects the bot instance into the CGI environment."""
```

Comment:
Noticed this wee typo while skimming through to decide whether to put this in 0.2.0 or 0.3.0.

```suggestion
    """WSGI Middleware that injects the bot instance into the CGI environment."""
```